tests/unit/testServiceConfiguration.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+var test = require('tape');
+var lodash = require('lodash');
+var isNumber = lodash.isNumber;
+var Configuration = require('../fixtures/configuration.js');
+var level = process.env.GCLOUD_DEBUG_LOGLEVEL;
```

Comment:
Fixed to use `GCLOUD_ERRORS_LOGLEVEL` instead of `GCLOUD_DEBUG_LOGLEVEL`.  I also fixed this typo in the `tests/unit/testConfiguration.js` file.